core/src/main/java/de/evoila/cf/broker/service/impl/AsyncDeploymentServiceImpl.java

```diff
@@ -64,6 +70,9 @@ public void asyncDeleteInstance(DeploymentServiceImpl deploymentService,
 			ServiceInstance serviceInstance, Plan plan, PlatformService platformService, String jobProgressId) {
         JobProgress jobProgress = progressService.startJob(jobProgressId, serviceInstance.getId(),
                 "Deleting service..", JobProgress.DELETE);
+        if (jobProgress == null) {
+            return;
+        }
```

Comment:
Is this check at the correct location? In the other two methods, jobProgress was checked here since it was used in the catch block and after the try block.

Additionally, why is there no succeedProcess call for the delete method?

Reply:
This Nullcheck has been replaced with proper exception handling in this PR: https://github.com/evoila/osb-core/pull/98

Please review the changes, as I fixed the tests according to the new behaviour too.

Because the JobProgress object gets deleted from the database. According to spec we need to return StatusCode 410 Gone when fetching the calling last_operation anyway.